certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "1.15.0",
+		"title": "RAPID Platform (V1)",
+		"description": "The RAPID Platform connector allows you to interact with your RAPID sites to help automate your business.",
+		"license": {
+			"name": "RAPID Platform License"
+		},
+		"contact":{
+			"name": "RAPID Support",
+			"url": "http://www.rapidplatform.com.au",
+			"email": "support@rapidplatform.com.au"
+		}
+	},
+	"host": "api-test.rapidplatform.com",
```

Comment:
@MattFranklin-Rapid We do not certify the connectors with test data or test API URLs. Please update it to production one.

Reply:
Error on my part, updated to point to route to production